friByte.capture-the-flag.service/friByte.capture-the-flag.service/Controllers/LeaderboardController.cs

```diff
@@ -8,7 +8,7 @@ namespace friByte.capture_the_flag.service.Controllers;
 /// <summary>
 /// API endpoints for serving leaderboard
 /// </summary>
-[Authorize(Roles = $"{IdentityRoleNames.TeamRoleName}, {IdentityRoleNames.AdminRoleName}")]
+//[Authorize(Roles = $"{IdentityRoleNames.TeamRoleName}, {IdentityRoleNames.AdminRoleName}")]
```

Comment:
I think it's perfectly valid to keep this public forever

Reply:
Alrighty!